packages/amplify_core/lib/src/category/amplify_auth_category.dart

```diff
@@ -1355,6 +1355,37 @@ class AuthCategory extends AmplifyCategory<AuthPluginInterface> {
         () => defaultPlugin.rememberDevice(),
       );
 
+  /// {@template amplify_core.amplify_auth_category.fetch_current_device}
+  /// Retrieves the current device.
+  ///
+  /// For more information about device tracking, see the
+  /// [Amplify docs](https://docs.amplify.aws/lib/auth/device_features/q/platform/flutter/).
```

Comment:
Link this to the [gen 2 docs](https://docs.amplify.aws/flutter/build-a-backend/auth/manage-users/manage-devices/). I guess we'll also have to change these across the board later.

Reply:
[done](https://github.com/aws-amplify/amplify-flutter/pull/5070/commits/b52049328a3a23bc02aae109ec21c6cdc455f73f)